scipy/sparse/linalg/_onenormest.py

```diff
@@ -8,7 +8,7 @@
 __all__ = ['onenormest']
 
 
-def onenormest(A, t=2, itmax=5, compute_v=False, compute_w=False):
+def onenormest(A, t=2, itmax=5, compute_v=False, compute_w=False, *, rng=None):
```

Comment:
This might need a discussion forum post. It is arguably a bug fix, but there might be other ways to go about it (e.g. make the behavior inherently deterministic somehow.)